src/Transport/Topology/MetaModel/NamespaceInfo.cs

```diff
@@ -21,7 +21,7 @@ public NamespaceInfo(string name, string connectionString)
         public bool Equals(NamespaceInfo other)
         {
             return other != null
-                   && Name.Equals(other.Name)
+                   && Name.Equals(other.Name, StringComparison.InvariantCultureIgnoreCase)
```

Comment:
`StringComparison.InvariantCultureIgnoreCase` should be replaced with `StringComparison.OrdinalIgnoreCase`